### DISCUSSION TITLE
ValueError: LanceError(IO): KNNFlatExec node: query column vector is not a vector

### DISCUSSION THREAD

**Discussion by jasonpump** [lancedb/lancedb#495] [General]
I go the following error when trying to query the table with this query (**tbl.search(query_embedding).limit(1).to_df()**):
ValueError: LanceError(IO): KNNFlatExec node: query column vector is not a vector
here query _embedding is a list of double values. And the table has the following schema:
schema = pa.schema([
    pa.field("vector", pa.list_(pa.float64(), list_size=512)),
    pa.field("a", pa.string()),
    pa.field("b", pa.string()),
    pa.field("c", pa.string()),
    pa.field("d", pa.string()),
    pa.field("e", pa.string())
])

Any ideas on the problem?

**Comment by eddyxu** [comment]
Hey, @jasonpump , Lance vector column must be a float32 fixed size array right now. 

We should improve the error message to make it clear.

**Reply by jasonpump** [reply]
yea, I figured it out by myself, thanks, needs better documentation.
One not related question: does lancedb support hybrid search?
